appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/model/progress/Draggable.kt

```diff
@@ -12,5 +12,5 @@ interface Draggable : HasDefaultAnimationSpec<Float> {
 
     fun onDrag(dragAmount: Offset, density: Density)
 
-    fun onDragEnd()
+    fun onDragEnd(onSettled: (() -> Unit)? = null)
```

Comment:
The naming couples the concept of dragging to settling, which isn't necessarily true. If we implement free drag, it will not need any settling on the model for example.